src/libs/actions/Report.ts

```diff
@@ -4087,7 +4092,9 @@ function searchForReports(searchInput: string, policyID?: string) {
         },
     ];
 
-    const searchForRoomToMentionParams: SearchForRoomsToMentionParams = {query: searchInput, policyID: policyID ?? '-1'};
+    // API expects a string, that's why policyID must default to a string
+    // eslint-disable-next-line rulesdir/no-default-id-values
+    const searchForRoomToMentionParams: SearchForRoomsToMentionParams = {query: searchInput, policyID: policyID ?? ''};
```

Comment:
Same question here

Reply:
1. Should it be possible to call this request without a valid `policyID`?
2. If yes - can we pass undefined instead of '' string?
3. If no - add a condition to prevent it

Reply:
@VickyStash I'm not sure what the API expects or how to test it, but currently we send '-1' in case `policyID ` is `undefined`

Reply:
Answered here: https://github.com/Expensify/App/pull/54306#discussion_r1893603705